### PR TITLE
Fix issue #10080 - Installation fails if password contains special ch…

### PR DESCRIPTION
**Pull request by tusharnain4578**
Fix issue #10080 - Installation fails if password contains special characters

## Issue Reference
#10080

## Description
Installation fails if password contains special characters

## How To Test This?
install with php artisan bagisto:install, on database password step, input like my#password

### CODE REVIEW

File: packages/Webkul/Installer/src/Console/Commands/Installer.php
Comment (amit-webkul) [inline packages/Webkul/Installer/src/Console/Commands/Installer.php:357]: @tusharnain4578 ,

I think we don't need these changes because when we try to enter a password invalid or with a special character, the MySQL connection will not be created and it will always give an error to us.

**Review by amit-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Installer/src/Console/Commands/Installer.php
Comment (tusharnain4578) [reply packages/Webkul/Installer/src/Console/Commands/Installer.php:357]: @amit-webkul The special characters are allowed in mysql secure passwords, and in most of the sql server generated a strong password with special characters (if auto generated) which includes '#' also, which is causing issues in .env file as if string containing '#' is not enlosed in quotes, the text after it will be treated as comment.

**Review by amit-webkul — DISMISSED** [review]

**Comment by tusharnain4578** [comment]
@amit-webkul refactored the code, please review and merge.

**Review by amit-webkul — APPROVED** [review]

**Comment by tusharnain4578** [comment]
@amit-webkul  Can you help by requesting another reviewer for this PR? We need a second review to proceed with the merge.

**Pull request merged**
